source/frontend/src/components/common/SectionListHeader.tsx

```diff
@@ -47,7 +47,7 @@ export const SectionListHeader: React.FunctionComponent<
 };
 
 const StyledRow = styled(Row)`
-  display: grid;
+  display: flex;
   grid-template-columns: 1fr 0.5fr;
```

Comment:
I'm fine with the change from grid to flex. However, the next line "grid-template-column" only applies to grids - wither restore the display grid or delete this line. Also please make sure that the layout is not broken with this change. thanks!